src/satosa/routing.py

```diff
@@ -170,6 +186,8 @@ def endpoint_routing(self, context):
         try:
             name, frontend_endpoint = self._find_registered_endpoint(context, self.frontends)
         except ModuleRouter.UnknownEndpoint:
+            for frontend in self.frontends.values():
+                logger.debug(f"Unable to find {context.path} in {frontend['endpoints']}")
```

Comment:
The `frontend['endpoints']` is a list of tuple where the first item is the regexp and the second id the registered callback. I would filter the `frontend['endpoints']` to show only the regexp's:
```python
f"Unable to find {context.path} in {[regexp for regexp, _ in frontend['endpoints']]}"
```